.travis.yml

```diff
@@ -17,6 +17,7 @@ script:
   # build test crates
   - cargo build -p multifile
```

Comment:
I think all these lines could be replaced with `cargo build --all`.

Reply:
Didn't know we could do that!